samples/multiplatform/androidApp/gradlew

```diff
@@ -0,0 +1,172 @@
+#!/usr/bin/env sh
```

Comment:
Better reuse shared gradlew in `../../`.

Reply:
Android Studio generates this wrapper while creating the project. I think it's better to not remove it because we have two separate builds in any case. The first one is the multiplatform library and is managed manually. The second one is the Android Studio build which compiles the Android app and is fully managed by Android Studio (including Gradle's version and its updates).

The Gradle guys also create a separate wrapper for each sample project in their samples (e.g. https://github.com/gradle/native-samples/tree/master/cpp/application)